docker/ts_config/timesketch.conf

```diff
@@ -0,0 +1,12 @@
+DEBUG = False
+SECRET_KEY = u'dockerdockerdocker'
```

Comment:
Right, I had that there for testing.
The entrypoint also adds a demo user, which you also might want to remove...
But better push something so we get somewhere...
(not working in this commit: celery worker)